Makefile

```diff
@@ -53,12 +53,12 @@ bin/ronn bin/cucumber:
 fmt:
 	go fmt ./...
 
-man-pages: $(HELP_ALL:=.ronn) $(HELP_ALL) $(HELP_ALL:=.txt)
+man-pages: share/man/man1/hub.1 $(HELP_ALL:=.ronn) $(HELP_ALL:=.txt)
 
 %.txt: %.ronn
 	groff -Wall -mtty-char -mandoc -Tutf8 -rLL=$(TEXT_WIDTH)n $< | col -b >$@
 
-%.1: %.1.ronn bin/ronn
+%.1: %.1.ronn bin/ronn $(HELP_ALL:=.ronn)
```

Comment:
You're right, this is not as straightforward as before.

What's the overall benefit of this change? On my computer, I don't see an improvement in your branch runing `make man-pages`, and with running `make -j2 man-pages` I see a 1s improvement. I'm not sure if supporting the parallelism case is worth it.

How often are you building man-pages? Is the current speed unacceptable?

Reply:
I only see it run one time, since a single `bin/ronn` invocation is used to pass in all `share/man/man1/*.ronn` files

```
$ git clean -fdxq share; time make -d man-pages 2>/dev/null | grep ^bin/ronn
bin/ronn --organization=GITHUB --manual="Hub Manual" share/man/man1/*.ronn

real    0m3.109s
user    0m2.102s
sys     0m2.024s
```

Reply:
> try
> 
> ```shell
> git clean -fdxq share ; rm -f bin/hub ; time make -j2 man-pages 2>/dev/null | grep ^bin/ronn
> ```

Ah yes, thank you for that. I can see the problem now. Is this something that people will likely encounter, though? I see this as an edge case that might not be trivial for us to fix. We've already spent a disproportionate amount of time discussing and debugging this when compared to the actual impact it has on our users.

Reply:
I've seen it in our package builds (because people think, using parallelism on today's 32-core systems is a good thing)
https://build.opensuse.org/package/live_build_log/openSUSE:Factory/hub/standard/x86_64

I can experiment some more with make to come up with something nicer.